tests/src/rules/no-absolute-path.js

```diff
@@ -53,48 +55,76 @@ ruleTester.run('no-absolute-path', rule, {
   invalid: [
     test({
       code: 'import f from "/foo"',
+      filename: '/foo/bar/index.js',
+      errors: [error],
+      output: 'import f from ".."',
+    }),
+    test({
+      code: 'import f from "/foo/bar/baz.js"',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'import f from "./baz.js"',
     }),
     test({
       code: 'import f from "/foo/path"',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'import f from "../path"',
     }),
     test({
       code: 'import f from "/some/path"',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'import f from "../../some/path"',
     }),
     test({
       code: 'import f from "/some/path"',
+      filename: '/foo/bar/index.js',
       options: [{ amd: true }],
       errors: [error],
+      output: 'import f from "../../some/path"',
     }),
     test({
       code: 'var f = require("/foo")',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'var f = require("..")',
     }),
     test({
       code: 'var f = require("/foo/path")',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'var f = require("../path")',
     }),
     test({
       code: 'var f = require("/some/path")',
+      filename: '/foo/bar/index.js',
       errors: [error],
+      output: 'var f = require("../../some/path")',
     }),
     test({
       code: 'var f = require("/some/path")',
+      filename: '/foo/bar/index.js',
       options: [{ amd: true }],
       errors: [error],
+      output: 'var f = require("../../some/path")',
     }),
     // validate amd
     test({
       code: 'require(["/some/path"], function (f) { /* ... */ })',
+      filename: '/foo/bar/index.js',
       options: [{ amd: true }],
       errors: [error],
+      output:
+        'require(["../../some/path"], function (f) { /* ... */ })',
```

Comment:
```suggestion
      output: 'require(["../../some/path"], function (f) { /* ... */ })',
```
i smell prettier :-p